### ISSUE TITLE
Tweet could not be processed

### ISSUE THREAD

**Issue by tristanbes** [scottwb/jekyll-tweet-tag#9]
`{"errors":[{"message":"SSL is required","code":92}]}`

On local environment

**Comment by aminalhazwani** [comment]
I get the message on local environment too. I am running just jekyll, no octopress. Terminal prints no errors but inside the browser the I get "Tweet could not be processed"

**Comment by aminalhazwani** [comment]
@tristanbes [this tweet plugin](https://github.com/kzykbys/JekyllPlugins/blob/master/tweet.rb) works like a charm

**Comment by tristanbes** [comment]
Thanks @aminalhazwani , anyway, i realized that running jekyll on github does not allow to run plugins ;-)

**Comment by aminalhazwani** [comment]
Yes, because GitHub pages run in safe-mode. In order to use 3rd party plugins you must host your website somewhere else.

**Comment by rob-murray** [comment]
Hey, This project doesn't seem to be maintained at the moment and is broken since Twitter updated their API to require ~~Oauth~~ SSL.

I have created a new version here: https://github.com/rob-murray/jekyll-twitter-plugin

Uses a slightly different tag name.

**Comment by nickdesaulniers** [comment]
Nice! Thanks @rob-murray !
